docs/guide/samples-environments.md

```diff
@@ -28,6 +28,23 @@ This means that a robot could open a drawer, leave an object inside it, and clos
 Light objects such as the books on the shelf or the fruits in the basket can be grabbed by a robot.
 An e-puck robot patrols on the table and an iRobot Create robot cleans the ground.
 
+### [complete\_apartment.wbt](https://github.com/omichel/webots/tree/master/projects/samples/environments/indoor/worlds/complete_apartment.wbt)
+
+**Keywords**: Apartment, domestic environment, furniture
```

Comment:
`domestic` has connotations and `domestic environment` is a business term. What about `household` instead?

```suggestion
**Keywords**: Apartment, household environment, furniture
```